Chore: docs updates (#46)
vercel/swr-site: pages/docs/mutation.md

## Mutate Based on Current Data

> Sometimes, you want to update a part of your data. So you need to get the current data first for the mutation.

Reviewer: ```suggestion
Sometimes, you want to update a part of your data based on the current data.
```
